plan/task/clientfilesystemread.go

```diff
@@ -35,6 +35,8 @@ func (t clientFilesystemReadTask) PreRun(_ context.Context, pctx *plancontext.Co
 		return fmt.Errorf("path %q is not a directory", path)
 	case pi.IsDir() && !isFS:
 		return fmt.Errorf("path %q cannot be a directory", path)
+	case err != nil:
+		return fmt.Errorf("path %q cannot be stat'd", path)
```

Comment:
Raise original error so user has a chance to fix it (e.g. permission error or whatnot):

```
return fmt.Errorf("path %q cannot be stat'd: %w", path, err)
```

Reply:
Right, I forgot to wrap the error. The other cases didn't have any error to wrap…